### PR TITLE
python3: fix chartype is unstable

### PR DESCRIPTION
**Pull request by methane**
Python's dict is unstable order.
It makes unstable output.

### CODE REVIEW

File: benchmark/tinysegmenter3.py [hunk 1]
```diff
@@ -21,16 +22,16 @@
...
-    "[0-9０-９]":"N"
-}
+_patterns = [
+    ("[一二三四五六七八九十百千万億兆]", "M"),
+    ("[一-龠々〆ヵヶ]", "H"),
+    ("[ぁ-ん]", "I"),
+    ("[ァ-ヴーｱ-ﾝﾞｰ]", "K"),
+    ("[a-zA-Zａ-ｚＡ-Ｚ]", "A"),
+    ("[0-9０-９]", "N"),
+]
```
Comment (chezou) [inline benchmark/tinysegmenter3.py:32]: Array of tuple! :eyes:

Comment (methane) [inline benchmark/tinysegmenter3.py:32]:
```
-    "[一-龠々〆ヵヶ]":"H",
```

may wrong. See https://github.com/chezou/TinySegmenter.jl/blob/master/src/TinySegmenter.jl#L67

**Comment by chezou** [comment]
Thank you for PR :+1:

File: benchmark/tinysegmenter3.py (hunk 1 above)
Comment (chezou) [inline benchmark/tinysegmenter3.py:32]: In julia code,  "H" flag is caught by L67 and L86
https://github.com/chezou/TinySegmenter.jl/blob/master/src/TinySegmenter.jl#L86

So, this python code is not wrong.

Comment (methane) [inline benchmark/tinysegmenter3.py:32]: I see.  This code can be faster.  I'll send another PR.

**Pull request merged**